shared/message/platform/NUSenseData.proto

```diff
@@ -102,4 +110,5 @@ message NUSense {
      */
     map<uint32, Servo> servo_map = 1;
     IMU                imu       = 2;
+    string             dispatch  = 3;
```

Comment:
Can I have a little bit of docs here please? IMO servo_map and imu is quite intuitive but what is dispatch supposed to be? Do we really want to log this message for every single NUSense packet

Reply:
We could make it its own standalone packet, but for now, I want it to at least work with what we have now which is only the one NUSense packet.

The dispatch a generic string message for uncommon errors and exceptions, e.g. nanopb.

Reply:
Added a comment.

Reply:
I wouldn't muddy this with an extra field for this. If you want to have specific log messages, make a new log message type and send that

Reply:
Will remove the `dispatch` field and move it to another message in a different PR.